Makefile

```diff
@@ -2,7 +2,8 @@ lint:
 	golangci-lint run --config golangci.yaml
 
 pre-pr: lint
-	$(MAKE) -C ./example run
+	$(MAKE) -C ./example clean run
```

Comment:
Since we tend to use pre-pr as a check that things are working, I figured we should probably clean up the project before running so that we don't have things succeeding via a cached version.